File: Cpp_Course/Cpp_Course/Files/better_calculator.c
#include <iostream>

using namespace std;

int main()
{
    int num1, num2;
    char op;

    cout << "Enter first number: " << endl;
    cin >> num1;
    cout << "Which operation do you want to carry out? Enter +,-,* or /:" << endl;
    cin >> op;
    cout << "Enter second number; " << endl;
    cin >> num2;

    int result;

    if(op == '+'){
        result = num1 + num2;
    } else if(op == '-'){
        result = num1 - num2;
    } else if(op == '*'){
        result = num1 * num2;
    } else if(op == '/'){
        result = num1 / num2;
    } else {
        cout << "Invalid operator" << endl;
    }

    cout << "The result is: " << result << endl;

    return 0;
}

File: Ruby_Course/Basics/better_calculator.rb
puts "Enter first number: "
num1 = gets.chomp().to_f
puts "Enter operator (mathematical SIGN!): "
op = gets.chomp()
puts "Enter second number: "
num2 = gets.chomp().to_f

def calculator(op, num1, num2)
  if op == "+"
    result = num1 + num2
  elsif op == "-"
    result = num1 - num2
  elsif op == "*"
    result = num1 * num2
  elsif op == "/"
    result = num1 / num2
  else
    puts "Unknown operator!"
    return
  end
  puts ("The result after " + op + " operation is: " + result.to_s + ".")
end

calculator(op, num1, num2)
